triggers/triggers.go

```diff
@@ -52,8 +53,27 @@ func GetTriggers(templatesCfg []NotificationTemplate, triggersCfg []Notification
 	return parseTriggers(triggersCfg, templates)
 }
 
+func spawnExprEnvs(opts map[string]interface{}) interface{} {
+	envs := map[string]interface{}{
+		"toTime": func(timestamp string) time.Time {
```

Comment:
If we introduce `time` namespace I would suggest renaming `toTime` to `parse`, so user would use it like `time.parse(...)`

Reply:
What a great idea. I will update this MR in a few hours.